.bumpversion.cfg

```diff
@@ -1,7 +1,7 @@
 [bumpversion]
 commit = False
 tag = True
-current_version = 1.0.812
+current_version = 1.0.813
```

Comment:
I thought that's not supposed to be part of the PR (it happens automatically)?

Reply:
How did it get to this point?

Reply:
Do you know what went wrong?  Why would a failed build require you to manually bump the version?

Reply:
Do u want me to open a separate PR ?

Reply:
Open new PR